### PR TITLE
Updates to latest Brave

### PR DESCRIPTION
**Pull request by adriancole**
Thanks for the investigation by @timtebeek

### CODE REVIEW

**Comment by adriancole** [comment]
https://github.com/apache/incubator-zipkin-brave/pull/927 fixes some of this

**Comment by adriancole** [comment]
I got a clean run with the bugfix noted in brave. I will cut a new release. sorry about the regression!

**Comment by timtebeek** [comment]
No problem at all! Only affected #1365 as Sleuth itself wasn't using 5.6.4 yet. Thanks for the quick fix (on a Saturday even).

File: pom.xml [hunk 1]
```diff
@@ -259,7 +259,7 @@
 		<spring-cloud-netflix.version>2.2.0.BUILD-SNAPSHOT</spring-cloud-netflix.version>
 		<spring-cloud-openfeign.version>2.2.0.BUILD-SNAPSHOT
 		</spring-cloud-openfeign.version>
-		<brave.version>5.6.1</brave.version>
+		<brave.version>5.6.4</brave.version>
```
Comment (timtebeek) [inline pom.xml]:
```suggestion
		<brave.version>5.6.5</brave.version>
```

Comment (adriancole) [reply pom.xml]: thx. there were actually 2 places with this version, I later found out. hopefully all good now!

**Comment by adriancole** [comment]
will merge on green so you can rebase your PR @timtebeek

**Comment by timtebeek** [comment]
Perfect, thanks a lot! :)

**Comment by adriancole** [comment]
cherry-picked into 2.0.x and 2.1.x after running tests locally

**Pull request merged**